### ISSUE TITLE
When can I upgrade the version of the component I depend on, such as electron?

### ISSUE THREAD

**Issue by zwdgit** [SimulatedGREG/electron-vue#927]

**Comment by ideacco** [comment]
Yes, I also want to know, because recently electron seems to be upgraded to 6.0.11, which solved many MAC system bugs.
It seems that vue-electron's project update is slow

**Comment by anribras** [comment]
I update to latest v6.0.12 electron but get error in `yarn dev`. Seems not work

**Comment by Stanzilla** [comment]
the project here is basically dead, there also is no maintained fork afaik

**Comment by linuxjuggler** [comment]
You can upgrade without any problem, my app is using electron 6 and the next version will use electron 7.

Check the issues as we already addressed this topic multiple times

**Comment by linuxjuggler** [comment]
This issue address the upgrade process for electron https://github.com/SimulatedGREG/electron-vue/issues/855

**Comment by umbrella22** [comment]
If you don't mind my semi-finished project, you can use my repository. I will sort out the English documents in the near future.
